pymove/core/dataframe.py

```diff
@@ -2082,6 +2111,13 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None, inplace=False):
         ----------
         https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.dropna.html
         """
+        
+        if inplace and (axis==1 or axis=='columns'):
```

Comment:
Se colocar, por exemplo, `move_df._data.dropna(axis=1, subset=[0])`, e na linha 0 a coluna `'lat'` for `NaN`, ele remove e coluna toda